### PR TITLE
Update Monitoring Docs

### PR DESCRIPTION
**Pull request by rasheedamir**
Signed-off-by: faizanahmad055 <faizan.ahmad55@outlook.com>

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-excercise:SNAPSHOT-PR-23-0e9f69c0`

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. `docker pull stakater/workshop-excercise:SNAPSHOT-PR-23-3b6b4c68`

**Comment by rasheedamir** [comment]
@faizanahmad055 plz follow same naming convention for images

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. `docker pull stakater/workshop-excercise:SNAPSHOT-PR-23-b25b62fd`

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. `docker pull stakater/workshop-excercise:SNAPSHOT-PR-23-d5f93393`

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. `docker pull stakater/workshop-excercise:SNAPSHOT-PR-23-3627ccc3`

File: docs/5-the-deployments-strike-back/1-autoscaling.md [hunk 1]
```diff
@@ -79,10 +79,8 @@
...
-    ![petbattle-api-hpa](./images/petbattle-api-hpa.png)
-    ![petbattle-api-hpa-topology](./images/petbattle-api-hpa-topology.png)
-    ![petbattle-api-deployment](./images/petbattle-api-deployment.png)
+    ![HPA_hey_command](./images/hpa-hey-command.png)
+    ![Pod_Scaled_Up](./images/pod-scaled-up.png)
+    ![HPA_Action1](./images/hpa_action1.png)
+    ![HPA_Action2](./images/hpa_action2.png)
 
-7. After a few moments you should see the autoscaler settle back down and the replicas are reduced.
-
```
Comment (rasheedamir) [inline docs/5-the-deployments-strike-back/1-autoscaling.md:87]: @faizanahmad055 why did you remove this part? we should keep it

**Comment by rasheedamir** [comment]
> With the change synchronized, we should see a new HorizontalPodAutoscaler object in ArgoCD and the cluster. Feel free to check those out.

@faizanahmad055 plz add screenshots for this section as well; and guide user how to find them

**Comment by rasheedamir** [comment]
> git commit -m  "🐎ADD - HPA enabled for test env 🐎 "

@faizanahmad055 isn't it dev environment and not test?

File: docs/5-the-deployments-strike-back/1-autoscaling.md (hunk 1 above)
Comment (rasheedamir) [reply docs/5-the-deployments-strike-back/1-autoscaling.md:87]: we should present the scale down as well
